webpack/src/Components/EmptyState/EmptyPageSelectors.js

```diff
@@ -0,0 +1,2 @@
+const selectEmptyState = state => state.pluginTemplate.emptyState;
```

Comment:
can you add another general selector of `const selectPluginTemplate = state => state.pluginTemplate` ?
and than you can take the `emptyState` key from it